tests/datasets/test_views.py

```diff
@@ -9,6 +9,80 @@
 from vitrina.datasets.factories import DatasetFactory, DatasetStructureFactory
 from vitrina.datasets.models import Dataset
 from vitrina.orgs.factories import OrganizationFactory
+from vitrina.resources.factories import DatasetDistributionFactory
+
+
+@pytest.fixture
+def dataset_detail_data():
+    organization = OrganizationFactory()
+    dataset = DatasetFactory(status="HAS_DATA", organization=organization)
+    dataset_distribution = DatasetDistributionFactory(dataset=dataset)
```

Comment:
```suggestion
    dataset_distribution = DatasetDistributionFactory(dataset__status="HAS_DATA")
```

Use data factories, instead of fixtures. Refactor data initialization into fixtures, only if there is a clear repeated case.

And there is no need to create all related objects, [factory boy](https://factoryboy.readthedocs.io/en/stable/reference.html#subfactory) does that automatically.